api/envoy/config/trace/v2alpha/xray.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package envoy.config.trace.v2alpha.xray;
+
+option java_outer_classname = "XRayTracer";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.config.trace.v2alpha";
+
+import "envoy/api/v2/core/base.proto";
+
+import "validate/validate.proto";
+
+// [#not-implemented-hide:]
```

Comment:
This is now implemented. Can you remove this, make sure the docs render nicely, and also add a release note about this new feature?